fv3config/_config.py

```diff
@@ -33,6 +38,16 @@ def get_default_config():
     return config
 
 
+def get_n_processes(config_dict):
```

Comment:
Standard variable name is ``config`` not ``config_dict``

More generally, is there a good reason to be passing the full config dict to functions that only need a small piece of them? We are doing this throughout the code, and it's made unit testing hard for me in certain cases. We can discuss offline or in a separate issue.